crates/bevy_text/src/text.rs

```diff
@@ -127,7 +52,7 @@ impl Text {
 }
 
 /// Contains the value of the text in a section and how it should be styled.
-#[derive(Debug, Default, Clone, Reflect)]
```

Comment:
Same on this component, should probably indicate that this component is required to be a child of the other Component.